2021-11-19 16:43:01.477 | WARNING  | autodidaqt_common.remote.schema:from_type:250 - Assuming naive implementation of Array and Object types.
2021-11-19 16:43:01.484 | INFO     | autodidaqt.core:start:768 - Application in startup.
2021-11-19 16:43:01.487 | INFO     | autodidaqt.core:start:769 - CommandLineConfig(headless=False, remote_config=None)
2021-11-19 16:43:01.490 | INFO     | autodidaqt.core:start:770 - {'version': '0.0.0', 'debug': True, 'logging': True, 'logging_directory': 'out\\logs', 'log_format': '{time}-{user}-{session}', 'save_format': 'pickle', 'data_directory': 'out\\data', 'data_format': '{user}\\{date}-{session}-{run}', 'maintain_state': True, 'state_directory': 'out\\state', 'instruments': {'simulate_instruments': False}, 'use_profiles': False, 'profiles': {}}
2021-11-19 16:43:01.495 | INFO     | autodidaqt.core:start:775 - Configuring Qt application and styles
2021-11-19 16:43:01.600 | INFO     | autodidaqt.core:configure_event_loop:746 - Configuring async runtime
2021-11-19 16:43:01.603 | INFO     | autodidaqt.core:configure_event_loop:753 - Using asyncio for async support.
2021-11-19 16:43:01.606 | INFO     | autodidaqt.core:configure_event_loop:763 - Setting custom exception handler on async loop
2021-11-19 16:43:02.052 | INFO     | autodidaqt.core:launch_panel:130 - Opening panel _instrument_manager
2021-11-19 16:43:02.057 | INFO     | autodidaqt.panels.instrument_manager:launch_panel:21 - Instrument Manager: Opening panel analyzer
2021-11-19 16:43:02.558 | INFO     | autodidaqt.panels.instrument_manager:launch_panel:21 - Instrument Manager: Opening panel phony
2021-11-19 16:43:02.773 | INFO     | autodidaqt.core:launch_panel:130 - Opening panel experiment
2021-11-19 16:43:02.823 | INFO     | autodidaqt.core:master:548 - Started async loop.
2021-11-19 16:43:02.825 | INFO     | autodidaqt.core:master:567 - Running actor .prepare
2021-11-19 16:43:02.827 | INFO     | autodidaqt.experiment.fsm:transition_to:42 - {'match': <ExperimentTransitions.Initialize: 'initialize'>, 'to': <ExperimentStates.Idle: 'IDLE'>}, initialize
2021-11-19 16:43:02.847 | INFO     | autodidaqt.core:master:584 - Installing Qt event processing on the standard event loop
2021-11-19 16:43:02.880 | INFO     | autodidaqt.core:load_state:690 - Loading application state.
2021-11-19 16:43:03.145 | INFO     | autodidaqt.core:load_state:694 - Found candidate application state: C:\Users\admin.TOF-PEEM\Documents\peem-daq\out\state\2021-11-19T16-36-29-479432.state.pickle... loading.
2021-11-19 16:43:03.175 | INFO     | autodidaqt.core:load_state:719 - Finished loading application state.
2021-11-19 16:43:03.236 | INFO     | autodidaqt.core:master:589 - Main task is dropping into event loop
2021-11-19 16:43:12.297 | ERROR    | autodidaqt.core:handle_exception:382 - Caught Unhandled: Failed in nopython mode pipeline (step: nopython frontend)
[1m[1mnon-precise type array(pyobject, 1d, C)[0m
[0m[1mDuring: typing of argument at C:\Users\admin.TOF-PEEM\Documents\peem-daq\daq\ether_daq\panel.py (25)[0m
[1m
File "daq\ether_daq\panel.py", line 25:[0m
[1mdef _acc_into_marginal(target_marginal, index_list, ir, ix, iy, low, high):
    <source elided>
    # step through the electron list and accumulate those in range
[1m    for i in range(index_list.shape[0]):
[0m    [1m^[0m[0m

2021-11-19 16:43:12.406 | WARNING  | autodidaqt.core:shutdown:417 - Shutting down due to exception.
2021-11-19 16:43:12.466 | INFO     | autodidaqt.core:shutdown:419 - Shutting down process pool.
2021-11-19 16:43:12.500 | INFO     | autodidaqt.core:shutdown:421 - Finished shutting down process pool.
2021-11-19 16:43:12.553 | INFO     | autodidaqt.core:shutdown:423 - Saving...
2021-11-19 16:43:12.582 | INFO     | autodidaqt.core:save_state:722 - Saving application state.
2021-11-19 16:43:12.612 | INFO     | autodidaqt.core:save_state:730 - Finished saving application state.
2021-11-19 16:43:12.650 | INFO     | autodidaqt.core:shutdown:427 - Application is still reading messages. Stopping...
2021-11-19 16:43:12.697 | INFO     | autodidaqt.core:shutdown:431 - Sending graceful shutdown requests.
2021-11-19 16:43:12.774 | ERROR    | autodidaqt.core:handle_exception:382 - Caught Unhandled: 'NoneType' object has no attribute 'put'
2021-11-19 16:44:15.980 | INFO     | autodidaqt.core:start:794 - Closed autodidaqt successfully.
